### PR TITLE
performance regression with user profiles test

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: performance_regression_test.py [hunk 1]
```diff
@@ -375,11 +367,24 @@ def test_mixed(self):
...
+        self.display_results(results, test_name='test_mixed')
         self.generate_stats_json(results, [base_cmd_w, base_cmd_m])
 
+    def test_user_profile(self):
+        """
+        Run workload using user profile(-s)
+        """
+        for cs_profile in self.params.get('sc_user_profiles').split():
+            self.log.debug('Run stress test with user profile {}'.format(cs_profile))
+            profile_dst = os.path.join('/tmp', os.path.basename(cs_profile))
```
Comment (amoskong) [inline performance_regression_test.py]: How do we prepare profiles to `/tmp` directory before testing? can we add them to SCT repo?

Comment (larisau) [inline performance_regression_test.py]: We'll use the profiles from scylla-qa-internal repo. I cannot define the full path in the yaml file, so I'll add cloning of the repo and replace file names with fullpath names in jenkins script.
If you'll have some better idea, please advise.

Comment (amoskong) [inline performance_regression_test.py]: Sounds good. How about check the files or directory exist here? Raise error and reminder user to prepare the profile.

Comment (larisau) [inline performance_regression_test.py]: you're right, the check is required, adding this

File: performance_regression_test.py [hunk 2]
```diff
@@ -375,11 +367,27 @@ def test_mixed(self):
...
+        Run workload using user profile(-s)
+        """
+        user_profiles = self.params.get('sc_user_profiles')
+        assert user_profiles is not None, 'No user profiles defined!'
+        for cs_profile in user_profiles.split():
+            assert os.path.exists(cs_profile), 'File not found: {}'.format(cs_profile)
+            self.log.debug('Run stress test with user profile {}'.format(cs_profile))
+            profile_dst = os.path.join('/tmp', os.path.basename(cs_profile))
+            for op in ('insert', 'get'):
+                stress_cmd = ("cassandra-stress user profile={} 'ops({}=1)' duration=10m -rate threads=100".format(
```
Comment (roydahan) [inline performance_regression_test.py]: Change to 60m

File: performance_regression_test.py [hunk 3]
```diff
@@ -375,11 +367,27 @@ def test_mixed(self):
...
+                    profile_dst, op))
+                stress_queue = self.run_stress_thread(stress_cmd=stress_cmd, stress_num=2, profile=cs_profile)
+                results = self.get_stress_results(queue=stress_queue, stress_num=2)
+                self.display_results(results, test_name='test_write')
+                self.generate_stats_json(results, [stress_cmd])
+
```
Comment (roydahan) [inline performance_regression_test.py]: Add "DB cleanup" (drop keyspace) between runs.

File: tests/gce-perf-regression-read-1.7-1hour.yaml [hunk 4]
```diff
@@ -7,6 +7,7 @@ failure_post_behavior: destroy
 space_node_threshold: 644245094
 ami_id_db_scylla_desc: '1.7.rc2'
 #ip_ssh_connections: 'private'
+sc_user_profiles: cust_a/features/doc1.yaml cust_a/offline/ctrs.yaml cust_b/events.yaml cust_c/group.yaml cust_e/events.yaml cust_s/case1.yaml cust_s/case2.yaml
```
Comment (roydahan) [inline tests/gce-perf-regression-read-1.7-1hour.yaml]: Check split method so we can have long lists.

**Pull request merged**